src/wing/CCPACSWingSection.h

```diff
@@ -37,32 +37,58 @@ class CCPACSWingSection : public generated::CPACSWingSection
 public:
     TIGL_EXPORT CCPACSWingSection(CCPACSWingSections* parent, CTiglUIDManager* uidMgr);
 
-    // Get element count for this section
+    /**
+    * @brief Get element count for this section
+    * @return int
+    */
     TIGL_EXPORT int GetSectionElementCount() const;
 
-    // Get element for a given index
+    /**
+    * @brief Get element for a given index
+    * @return CCPACSWingSectionElement
+    */
     TIGL_EXPORT CCPACSWingSectionElement& GetSectionElement(int index);
     TIGL_EXPORT const CCPACSWingSectionElement& GetSectionElement(int index) const;
 
-    // Gets the section transformation
+    /**
+    * @brief Gets the section transformation
+    * @return CTiglTransformation
+    */
     TIGL_EXPORT CTiglTransformation GetSectionTransformation() const;
 
-    // Gets the section translation
+    /**
+    * @brief Returns the translation of a wing section in CPACS-coordinates
+    * @return CTiglPoint
     TIGL_EXPORT CTiglPoint GetTranslation() const;
 
-    // Gets the section rotation
+    /**
+    * @brief Returns the rotation of a wing section in degree
```

Comment:
We could mention that the angles are Euler angles in xy'z'' convention (https://en.wikipedia.org/wiki/Euler_angles -> See Tait-Bryan angles, https://www.cpacs.de/documentation/CPACS_3_4_0_Docs/html/135055ee-b810-5e11-ed38-4e28a7017a29.htm)